app/models/form/page.rb

```diff
@@ -24,6 +24,12 @@ def initialize(id, hsh, subsection)
 
   delegate :form, to: :subsection
 
+  def header
+    return @header if @copy_key.nil?
+
+    I18n.t("forms.#{form.start_date.year}.#{@copy_key}.page_header")
+  end
```

Comment:
Would it make sense to set the @header to this to begin with? Would that make it a bit more performant?